### ISSUE TITLE
detail about your coding

### ISSUE THREAD

**Issue by amirazai123** [KristaRutz/oranga-app#23]
Can I know the detail about what you are trying to do?

**Comment by KristaRutz** [comment]
> Can I know the detail about what you are trying to do?

Sure, was this a general comment or about a specific section of the code?

**Comment by amirazai123** [comment]
Are you trying to do a detection burnout app or something like that?

**Comment by KristaRutz** [comment]
@amirazai123 Yes, Oranga is about helping people that might be burned out (or are on a path to burnout) identify how they feel! Our MVP is about raising the user’s awareness of their mental health levels and issues, and trying to identify what might be the specific cause behind their burnout.

Although production is paused on this application, our next plan is to develop education and accountability features, which are two roadblocks that young professionals face in combatting burnout and achieving healthy work-life balance. Basically, we know that it’s hard to stay consistent and motivated, and it becomes even more difficult when you don’t have clear results or foundations for your motives.

**Comment by amirazai123** [comment]
I asked you because im doing a burnout analytic for teacher for my final
year project.
Im having problem in classify burnout from maslach burnout inventory survey.
Do you have some tips about how to classify level and diagnose whether
someone have burnout or not from the survey?

**Comment by KristaRutz** [comment]
We allow for people to self-diagnose. If they don’t feel their best, and they suffer from common burnout problems, they are probably burned out! 
We used these factors from a UC Berkeley model published in Harvard Business Review in 2019 to determine what factors might be involved.
![79B934FA-AC96-469A-8F29-50299C14AB3A](https://user-images.githubusercontent.com/57574670/104866865-6da92c00-58f4-11eb-98fa-650a871a6358.jpeg)

**Comment by amirazai123** [comment]
Oh you did not use any offical burnout survey?
And what is the class label that you use in order to self diagnosis the
burnout syndrome?

**Comment by KristaRutz** [comment]
No one on our team was qualified to diagnose levels of burnout, which is why our questions are focused on raising awareness. Hopefully this information can help!

**Comment by amirazai123** [comment]
Did you only self diagnose or you also diagnose based on their level?
